pragma-node/src/types/ws/mod.rs

```diff
@@ -48,9 +50,14 @@ pub struct Subscriber<ChannelState> {
     pub receiver: SplitStream<WebSocket>,
     pub update_interval: Interval,
     pub notify_receiver: Receiver<Message>,
+    pub rate_limiter: DefaultKeyedRateLimiter<IpAddr>,
```

Comment:
I think this should be wrapped: `Option<DefaultKeyedRateLimiter<IpAddr>>`
And we should allow the user to pass the limit in the `new` function, again as an option:
```rust
pub async fn new(
    socket: WebSocket,
    ip_address: IpAddr,
    app_state: Arc<AppState>,
    state: Option<ChannelState>,
    update_interval_in_ms: u64,
    bytes_rate_limit_per_s: Option<u32>
) -> Result<(Self, Sender<Message>), WebSocketError> {
    // & if bytes_rate_limit_per_s is None don't build the DefaultKeyedRateLimiter
```

Reply:
it's fine to put it as a constant, there won't be any rate limit by default unless rate limiter is explicitly used in the implementation